src/util/collections.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub use fnv::FnvHashSet as HashSet;
+pub use fnv::FnvBuildHasher as BuildHasherDefault;
+// pub use fnv::FnvHashMap as HashMap;
+// pub use std::collections::hash_map::Entry as HashMapEntry;
+// pub use std::collections::hash_map::Values as HashMapValues;
+
+pub use ordermap::OrderMap as HashMap;
```

Comment:
there may be lots of changes if not using HashMap. We can do so first, then rename a proper name later. Maybe we can use Map directly.